pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -261,14 +275,24 @@ impl BlockReader for EphemeralFile {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::context::DownloadBehavior;
+    use crate::task_mgr::TaskKind;
     use crate::tenant::block_io::{BlockCursor, BlockReaderRef};
     use rand::{thread_rng, RngCore};
     use std::fs;
     use std::str::FromStr;
 
     fn harness(
         test_name: &str,
-    ) -> Result<(&'static PageServerConf, TenantId, TimelineId), io::Error> {
+    ) -> Result<
+        (
```

Comment:
Maybe at this point it might make sense to introduce a `Harness` struct instead of returning tuples. But this can also be a followup change.

Reply:
Yeah, not doing that here.